### PR TITLE
Updating the Mgt latest version in the tab-graph-toolkit Csharp sample

### PR DESCRIPTION
**Pull request by Harikrishnan-MSFT**

### CODE REVIEW

File: samples/tab-graph-toolkit/csharp/TabGraphToolkit/ClientApp/public/auth-start.html [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+            src="https://alcdn.msauth.net/browser/2.21.0/js/msal-browser.min.js"
+            integrity="sha384-s/NxjjAgw1QgpDhOlVjTceLl4axrp5nqpUbCPOEQy1PqbFit9On6uw2XmEF1eq0s"
+            crossorigin="anonymous">
+    </script>
+    <script type="text/javascript">
+        microsoftTeams.app.initialize().then(() => {
+            microsoftTeams.app.getContext().then(async (context) => {
+                // Generate random state string and store it, so we can verify it in the callback
+                var currentURL = new URL(window.location);
+                var clientId = currentURL.searchParams.get("clientId");
```
Comment (Prithvi-MSFT) [inline samples/tab-graph-toolkit/csharp/TabGraphToolkit/ClientApp/public/auth-start.html:21]: where are we passing client ID

Comment (Harikrishnan-MSFT) [reply samples/tab-graph-toolkit/csharp/TabGraphToolkit/ClientApp/public/auth-start.html]: image

File: samples/tab-graph-toolkit/csharp/TabGraphToolkit/ClientApp/src/components/Tab.tsx [hunk 2]
```diff
@@ -0,0 +1,132 @@
+import { useContext } from "react";
+import { TeamsFxContext } from "./Context";
+import React from "react";
+import { Agenda, Person, applyTheme,People,PeoplePicker,Tasks,Todo, PersonCard, ViewType } from "@microsoft/mgt-react";
+import { Button } from "@fluentui/react-components";
+
```
Comment (Pawank-MSFT) [inline samples/tab-graph-toolkit/csharp/TabGraphToolkit/ClientApp/src/components/Tab.tsx:6]: remove extra line

**Review by Pawank-MSFT — APPROVED** [review]
One small comments, rest ok.

File: samples/tab-graph-toolkit/csharp/TabGraphToolkit/ClientApp/src/components/Tab.tsx (hunk 2 above)
Comment (Harikrishnan-MSFT) [reply samples/tab-graph-toolkit/csharp/TabGraphToolkit/ClientApp/src/components/Tab.tsx:6]: Fixed

**Comment by Harikrishnan-MSFT** [comment]
Build Pipeline - 
https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/8375360778

image

**Review by Prithvi-MSFT — APPROVED** [review]

**Review by Pawank-MSFT — APPROVED** [review]
looks correct, Approving!

**Comment by Harikrishnan-MSFT** [comment]
Build Pipeline -
https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/8448355379
image

**Pull request merged**